tests/plugins/test_tvplayer.py

```diff
@@ -33,20 +33,18 @@ def test_can_handle_url(self):
     @patch('streamlink.plugins.tvplayer.HLSStream')
     def test_get_streams(self, hlsstream, mock_get_stream_data):
         mock_get_stream_data.return_value = {
-            "stream": "http://test.se/stream1"
+            "response": {"stream": "http://test.se/stream1", "drm": None }
```

Comment:
PEP8: whitespace before '}'

```suggestion
            "response": {"stream": "http://test.se/stream1", "drm": None}
```